### DISCUSSION TITLE
Best way to transform nullish to optional

### DISCUSSION THREAD

**Discussion by thobson** [open-circle/valibot#405] [Q&A] [answered]
Browsers can send me `{ }` or `{ a: null }`, but the libraries I use expect `{ a?: undefined }`

I'm looking for a generic way to transform v.nullish(schema) to v.optional(schema) if possible?

I thought about `v.transform(v.nullish(schema), (s) => s ?? undefined)`

**Comment by fabian-hiller** [comment, marked as answer]
Seems good to me
